src/main/java/org/sagebionetworks/web/client/view/DataAccessManagementView.java

```diff
@@ -1,15 +1,7 @@
 package org.sagebionetworks.web.client.view;
 
 import com.google.gwt.user.client.ui.IsWidget;
-import org.sagebionetworks.web.client.presenter.DataAccessManagementPresenter;
 
 public interface DataAccessManagementView extends IsWidget {
-  /**
-   * Set this view's presenter
-   *
-   * @param presenter
-   */
-  public void setPresenter(DataAccessManagementPresenter presenter);
-
-  public void render();
+  void render();
```

Comment:
The view never used the presenter. It was easily simplified to just a `render` method.